UnitTest/TestComponentCSharp_Tests.cs

```diff
@@ -1594,27 +1600,27 @@ public void CCWOfListOfManagedType()
         public void WeakReferenceOfManagedObject()
         {
             var properties = new ManagedProperties(42);
-            WinRT.WeakReference<IProperties1> weakReference = new WinRT.WeakReference<IProperties1>(properties);
-            weakReference.TryGetTarget(out var propertiesStrong);
+            WeakRefNS::WeakReference<IProperties1> weakReference = new WeakRefNS::WeakReference<IProperties1>(properties);
```

Comment:
Are you sure this is supposed to be a double-colon instead of a period separator here?

Reply:
Both do seem to build, but I flipped it as . seems to be more common.